core/opl/cluster_read.py

```diff
@@ -178,6 +179,8 @@ def _sanitize_target(self, target):
         target = target.replace("$Cloud", self.args.grafana_prefix)
         return target
 
+    # pylint: disable-next=undefined-variable
+    @retry(stop=(stop_after_delay(10) | stop_after_attempt(10)))  # noqa: F405
```

Comment:
BTW never used `|` in python, thanks! https://stackoverflow.com/questions/5988665/pipe-character-in-python

Reply:
:)